src/browserless.ts

```diff
@@ -570,6 +571,19 @@ export class BrowserlessServer {
     return this.webdriver.proxySession(req, res);
   }
 
+  private onStart() {
+    debug(`Starting new job`);
+    const currentlyRunning =
+      this.queue.length >= this.queue.concurrencySize
+        ? this.queue.concurrencySize
+        : this.queue.length;
+
+    this.currentStat.maxConcurrent =
```

Comment:
@joelgriffith does this counter reset?  I was looking over the calls to `resetCurrentStat` and callers of `new BrowserlessServer`, and it appears the stats object is initialized once.   This means the counter will track the peak for the duration of the server instance.   Is that correct?

So, if I have a server running that peaks at 20 sessions, and then sits idle for hours it'll keep reporting 20.  Is this desired?  I'd expect the peak would be cleared when the metric is dumped out.  This is particularly important if you're aggregating session counters across multiple server instances.    ServerA could peak to 20 at say 6pm - 6:30pm, and serverB could peak to 20 at 6:30pm to 7pm.    Assuming they are idle outside of these times, the peak for 6pm - 7pm should be 20, not 40.  If the counters reset when dumped we could then take `Max(serverA counts) + Max(server B counts)`.  Thanks.

Reply:
Excellent, thanks.